data/json/items/melee/bludgeons.json

```
@@ -446,7 +446,7 @@
     "volume": "30 ml",
     "longest_side": "20 cm",
     "cutting": 6,
-    "to_hit": { "grip": "weapon", "length": "hand", "surface": "line", "balance": "good" },
+    "to_hit": { "grip": "solid", "length": "hand", "surface": "line", "balance": "good" },
```

```suggestion
    "to_hit": { "grip": "bad", "length": "hand", "surface": "line", "balance": "good" },
```

Grip sounds poor, based on the description.

---

// one of "bad", "none", "solid", or "weapon"
I was thinking that even wrapped it should still be odd shaped and hard to hold. Which of the above were you thinking?

---

I misunderstood the meaning of the JSON values. Indeed, a glass shiv with no wrapping would be "bad", and having a wrapping perhaps should upgrade it to "none". I would not consider it "solid".